spec/inputsOutputs/inputsOutputs.spec.js

```diff
@@ -72,13 +71,17 @@ describe('inputsOutputs page', function() {
 
     });
   });
+
   describe('remove input and output', function() {
 
     it('should remove input', function(done) {
       // remove input
       components.inputsOutputs.page.setInputOrOutputFields(inputElement, 'input');//fill fields
       components.inputsOutputs.page.submitInputOrOutput(inputElement);//add
       components.inputsOutputs.page.deleteInputOrOutput(inputElement, 'input');//remove element
+      expect((element(by.css('.popover')).isDisplayed())).toBeTruthy();//popover should show up
```

Comment:
move checking is popover present to a component, this code is repeated many times throughout the code.